core/src/encodeURL.ts

```diff
@@ -31,18 +31,34 @@ export interface TransferRequestURLFields {
     message?: Message;
     /** `memo` in the [Solana Pay spec](https://github.com/solana-labs/solana-pay/blob/master/SPEC.md#memo). */
     memo?: Memo;
+    /** `redirect` in the [Solana Pay spec](https://github.com/solana-labs/solana-pay/blob/master/SPEC1.1.md#redirect). */
+    redirect?: Redirect;
+}
+
+/**
+ * Fields of a Solana Pay message sign request URL.
+ */
+export interface MessageSignRequestURLFields {
```

Comment:
I think this type can be (structurally) the same as `TransactionRequestURLFields`, right? Label and message can still be used.

Reply:
I think maybe we should rename the type of request to a `Sign Message Request` in the spec and here.

Reply:
They can still be used, but it seems a bit confusing because wallets would presumably ignore them. Both fields are not inputs in the spec, and are also fields in server responses which I think makes it more confusing. We do already have this problem with transaction requests - I just wasn't sure whether it was a good idea to do the same here, and thought it might be better to move toward removing the fields from transaction request in the SDK instead.